File: DataAccess/Data/Models/Comment.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DataAccess.Data.Models
{
    public partial class Comment
    {
        public int Id { get; set; }

        public int ArticleId { get; set; }

        public Article Article { get; set; }

        public string Content { get; set; }

        public string Nickname { get; set; }

        public DateTime PublishDate { get; set; }

        public Comment()
        {

        }
    }
}

File: InfluencersApp/wwwroot/js/comment.js
$(document).ready(() => {

    $("#send").click(() => {

        let articleId = $("#article-id").html();

        var data = {
            ArticleId: parseInt(articleId),
            Nickname: $("#nickname").html(),
            Content: $("#comment").html(),
            Succeded: false
        }

        $.ajax({
            headers: {
                'Accept': 'application/json',
                'Content-Type': 'application/json'
            },
            type: "POST",
            data: JSON.stringify(data),
            success: (response) => {
                if (response.succeded == true) {

                }
                else {
                    alert("Oops, unable to add article... please try again!");
                }
            }
        });
    });
});